recipe/build.sh

```diff
@@ -2,5 +2,6 @@ cmake $CMAKE_ARGS \
 	-DCMAKE_BUILD_TYPE=Release \
 	-DLSL_BUNDLED_PUGIXML=OFF \
 	-DLSL_UNIXFOLDERS=ON \
+	-DCMAKE_CXX_STANDARD=11 \
```

Comment:
Turns out this was the key change to fix osx x86_64

Reply:
Great find!!